sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client_async.py

```diff
@@ -31,28 +31,29 @@
 import os
 import logging
 import re
+import hashlib
+from app_configuration_preparer import AppConfigurationPreparer
+from app_configuration_client_async_preparer import AppConfigurationClientPreparer
 
 class AppConfigurationClientTest(AzureMgmtTestCase):
+    try:
+        name_prefix = "appconfig-" + hashlib.md5(os.environ['RUN_IDENTIFIER'].encode()).hexdigest()[-3:]
```

Comment:
Don't do that, use the random_name_enable instead in ALL your RG preparer:
https://github.com/Azure/azure-sdk-for-python/blob/2c9b3309648c2ba669ac04ac318eef2ea15632e2/sdk/keyvault/azure-keyvault-secrets/tests/test_secrets_async.py#L65

If your AppConfiguration mgmt has the same kind of randmness needed (like a storage account must be unique worldwide), please force it in your __init__:
https://github.com/Azure/azure-sdk-for-python/blob/2c9b3309648c2ba669ac04ac318eef2ea15632e2/sdk/keyvault/azure-keyvault-keys/tests/keys_preparer.py#L27